### PR TITLE
Add Azure Pipelines (close #200)

### LINKED ISSUE #200: Azure Pipelines support
Will this service supports Azure Pipelines? More and more GitHub repositories use Azure Pipelines now, such as Yarn and TypeScript.

### PR DESCRIPTION
**Pull request by g-plane**
This is my first PR to this repo, so I'm not familiar too much...

### CODE REVIEW

File: libs/live-fns/azure-pipelines.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+module.exports = async (organization, project, definition, branch = 'master') => {
+  const response = await got(`https://dev.azure.com/${organization}/${project}/_apis/build/status/${definition}?branchName=${branch}`, { json: false })
+  const contentType = response.headers['content-type']
+
+  if (!contentType.includes('image/svg+xml')) {
+    return {
+      subject: 'Azure Pipelines',
+      status: 'unknown',
+      color: 'blue'
```
Comment (amio) [inline libs/live-fns/azure-pipelines.js]: `'grey'` would be better for unknown state

**Comment by g-plane** [comment]
Maybe Azure DevOps doesn't provide JSON API at all. If you check the source code in Shields.io, it also fetches the SVG and parses it.

File: libs/live-fns/azure-pipelines.js (hunk 1 above)
Comment (g-plane) [reply libs/live-fns/azure-pipelines.js]: Now it was fixed.

**Comment by amio** [comment]
@g-plane Hmm, just curious :D

**Pull request merged**